.travis.yml

```diff
@@ -728,6 +728,7 @@ script:
     # Build and execute all examples.
     - if [ "${ALPAKA_CI_ANALYSIS}" == "OFF" -a "${ALPAKA_ACC_GPU_CUDA_ONLY_MODE}" == "OFF" ] ;then ./travis/compileExec.sh "example/bufferCopy" ./bufferCopy ;fi
     - if [ "${ALPAKA_CI_ANALYSIS}" == "OFF" -a "${ALPAKA_ACC_GPU_CUDA_ONLY_MODE}" == "OFF" ] ;then ./travis/compileExec.sh "example/helloWorld" ./helloWorld ;fi
+    - if [ "${ALPAKA_CI_ANALYSIS}" == "OFF" -a "${ALPAKA_ACC_GPU_CUDA_ONLY_MODE}" == "OFF" -a ${ALPAKA_CUDA_VER_MAJOR} > 7 -a ${ALPAKA_CUDA_VER_MINOR} > 0] ;then ./travis/compileExec.sh "example/helloWorldLambda" ./helloWorldLambda ;fi    
```

Comment:
I am no bash expert but the syntax is not supported: The builds say `/home/travis/build.sh: line 45: [: missing `]'` but did not get red (I do not know why).
You may need to use double parenthesis like I did in other places to make the arithmetic comparison work.